client/components/deposits-overview/next-deposit.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * Internal dependencies.
+ */
+import strings from './strings';
+import './style.scss';
+import Loadable from 'components/loadable';
+
+interface NextDepositProps {
+	isLoading: boolean;
+}
+/**
+ * Renders the Next Deposit details component.
+ *
+ * This component included the next deposit heading, table and notice.
+ *
+ * @param {NextDepositProps} props Next Deposit details props.
+ * @return {JSX.Element} Rendered element with Next Deposit details.
+ */
+const NextDepositDetails: React.FC< NextDepositProps > = ( {
+	isLoading,
+}: NextDepositProps ): JSX.Element => {
```

Comment:
Nit: the expected component props type `NextDepositProps` is inferred via the `React.FC< NextDepositProps >` type parameter, so we can be concise and remove the duplicate `NextDepositProps` expected via the props object.
```suggestion
const NextDepositDetails: React.FC< NextDepositProps > = ( {
	isLoading,
} ): JSX.Element => {
```

Reply:
Aha, I've just been chucking in types everywhere to try and appease the editor 😂 Thanks for the tip!